packages/web-app/src/routes/index.spec.js

```diff
@@ -52,12 +52,9 @@ describe('routes/index', () => {
     expect(mockUse).toBeCalledWith(`/${views.checkAndConfirm}`, checkAndConfirmDetails);
     expect(mockUse).toBeCalledWith(`/${views.reviewComplete}`, reviewComplete);
     expect(mockUse).toBeCalledWith(
-      `/questionnaires-for-review/check-and-confirm`,
+      `/questionnaire-check-and-confirm`,
       questionnaireCheckAndConfirm
     );
-    expect(mockUse).toBeCalledWith(
-      `/questionnaires-for-review/already-reviewed`,
-      questionnaireAlreadyReviewed
-    );
+    expect(mockUse).toBeCalledWith(`/questionnaire-already-reviewed`, questionnaireAlreadyReviewed);
   });
 });
```

Comment:
I can't see a test for reviewQuestionnaireSubmission on this page.

Reply:
Addressed via commit [0a79768](https://github.com/Planning-Inspectorate/back-office/pull/83/commits/0a797681b995faa4d2229ae2b0a99b192eaa6f5d)